src/lib.rs

```diff
@@ -299,11 +304,13 @@ impl Executor {
             let id = ID.fetch_add(1, Ordering::Relaxed);
 
             // Spawn the new thread.
+            #[allow(unused_variables)]
             if let Err(e) = thread::Builder::new()
```

Comment:
I would prefer just renaming `e` to `_e` in this case, rather than enabling this lint.

Reply:
Done.